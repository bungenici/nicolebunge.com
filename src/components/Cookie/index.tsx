import { Link } from 'gatsby';
import React from 'react';
import CookieConsent from 'react-cookie-consent';
import * as routes from '../../constants/routes';

const Cookie: React.FC = () => (
  <CookieConsent>
    Um die Webseite optimal gestalten und fortlaufend verbessern zu k√∂nnen, werden Cookies
    verwendet. Durch die weitere Nutzung der Webseite stimmen Sie der Verwendung von Cookies zu.{' '}
    <Link to={routes.PRIVACY.path}>{routes.PRIVACY.name}</Link>
  </CookieConsent>
);

export default Cookie;
