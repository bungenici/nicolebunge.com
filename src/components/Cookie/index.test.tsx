/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import renderer from 'react-test-renderer';
import Cookie from '.';

describe('Cookie component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Cookie />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
