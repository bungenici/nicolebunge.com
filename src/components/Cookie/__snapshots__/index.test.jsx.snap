// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cookie component renders correctly 1`] = `
<div
  className="cookieConsent "
  style={
    Object {
      "alignItems": "baseline",
      "background": "#353535",
      "bottom": "0",
      "color": "white",
      "display": "flex",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "left": "0",
      "position": "fixed",
      "width": "100%",
      "zIndex": "999",
    }
  }
>
  <div
    className=""
    style={
      Object {
        "flex": "1 0 300px",
        "margin": "15px",
      }
    }
  >
    Um die Webseite optimal gestalten und fortlaufend verbessern zu k√∂nnen, werden Cookies verwendet. Durch die weitere Nutzung der Webseite stimmen Sie der Verwendung von Cookies zu.
     
    <a
      href="/datenschutz"
    >
      Datenschutz
    </a>
  </div>
  <button
    className=""
    id=""
    onClick={[Function]}
    style={
      Object {
        "background": "#ffd42d",
        "border": "0",
        "borderRadius": "0px",
        "boxShadow": "none",
        "color": "black",
        "cursor": "pointer",
        "flex": "0 0 auto",
        "margin": "15px",
        "padding": "5px 10px",
      }
    }
  >
    I understand
  </button>
</div>
`;
